package.json

```diff
@@ -31,41 +31,42 @@
     "request-json": "0.5.5"
   },
   "devDependencies": {
-    "brunch": "2.6.5",
+    "brunch": "1.8.5",
     "coffee-script": "1.10.0",
     "coffeelint": "1.15.0",
     "cozy-url-sdk": "1.0.2",
     "jade2commonjs": "0.1.0",
     "mocha": "2.4.5",
     "nodemon": "1.9.1",
-    "npm-run-all": "1.7.0",
+    "npm-run-all": "3.1.0",
     "should": "8.3.0"
   },
   "scripts": {
-    "start": "node build/server.js",
-    "test": "cake --use-js tests",
-    "test:mocha-no-env": "mocha test --reporter spec --colors --globals setImmediate,clearImmediate --compilers coffee:coffee-script/register",
+    "build": "npm run build:server && npm run build:client && npm run build:views",
+    "build:client": "cd client && brunch build --production",
```

Comment:
@gregorylegarec I've just take a look at the `package.json` file, and brunch is installed as a dev dependency, so this command works using the local version (`npm run` extend `$PATH` var to local node_modules directory). Can you try again? I can't figure out why it doesn't work…